lib/helpers/wdio.conf.ejs

```diff
@@ -295,6 +295,42 @@ exports.config = {
     // afterSuite: function (suite) {
     // },
     /**
+     * Runs before a Cucumber feature
```

Comment:
Just to clarify, from poking around it seems that the Mocha/Jasmine ones are before/after Suite, Hook, and Test, is that correct?